### DISCUSSION TITLE
请问如何动态导出 Icon 组件？

### DISCUSSION THREAD

**Discussion by miss85246** [arco-design/arco-design#1950] [Q&A]
前端小白，如题，现在所有的 Icon 都是要拼写 Icon 的名字作为组件，导入后才能进行使用，有没有什么方式能够统一一个入口，然后只更改填入的文字就返回对应的 Icon 的方式？求告知 🥲

**Comment by yinkaihui** [comment]
https://codepen.io/yinkaihui/pen/vYVQEOp?editors=0010

可参考
